src/main/java/net/pms/configuration/DeviceConfiguration.java

```diff
@@ -85,7 +85,6 @@ public void inherit(RendererConfiguration ref) throws ConfigurationException {
 		filter = baseConf.filter;
 
 		// Initialize our internal RendererConfiguration vars
-		renderCache = new HashMap<>();
 		sortedHeaderMatcher = ref.sortedHeaderMatcher;
```

Comment:
@Nadahar thanks for the review. I think the comment is still relevant for the remaining vars `sortedHeaderMatcher` and `loaded`